### ISSUE TITLE
Error running timer 'ac-update-greedy'

### ISSUE THREAD

**Issue by CraneHuang** [clojure-emacs/ac-cider#27]
Usually, ac-cider works well in cider.However , I get the error "running timer 'ac-update-greedy'" sometimes.The following picture shows the detail:
![2015-01-12 12 27 55](https://cloud.githubusercontent.com/assets/5162826/5895609/a281b5c2-a55f-11e4-9440-e2d3ac2b81b6.png)
In addition,I use emacs 24.4 and cider 0.8.2.

**Comment by alexander-yakushev** [comment]
I used to see those a few times too. Is it reliably reproducable by you?

**Comment by CraneHuang** [comment]
No,it isn't.This occurs by accident.

**Comment by alexander-yakushev** [comment]
Well, it's not much I can do then. I don't know AC internals well enough. If you are someday able trim it down at least to a single reproducible example, I'll be glad to help.

**Comment by CraneHuang** [comment]
Thanks . If I find a reproducible example,I will tell you here.

**Comment by alexander-yakushev** [comment]
@CraneHuang By the way I remembered where I've seen this error. It happens to me if I try to complete in a buffer where CIDER connection is absent. That's not your case, I guess.

**Comment by CraneHuang** [comment]
Sometimes,I get the error when I launch the cider and type some codes.At the time,my cider should be connected.

**Comment by bbatsov** [comment]
Generally such errors occur when there's some unhandled error in the
middleware. Thought, I managed to find all of those, but I guess I
didn't... On such an error the client will wait for a few seconds for a
response and timeout. Guess we'll have to review carefully the code in the
completion middleware for potential problems.

**Comment by johnbendi** [comment]
I don't think this is related to cider. I started getting this error immediately after updating my ubuntu system. The error is occuring without even starting cider. In my case it's occuring where AC is in minor mode to java-mode.

**Comment by TimePower** [comment]
This problem also occurs in ac-js2, Don't know why :-(

**Comment by kasz** [comment]
The same problem happens to me after updating cider-nrepl to 0.12.0 version. Commit cc8efb877cb80475392860a478c985421a25acd5 fixed this problem for me. 
I think that more people could benefit from this fix, if new version of ac-cider would be released, with this commit included.

**Comment by alexander-yakushev** [comment]
Perhaps, you mean releasing it to melpa-stable. I tagged the latest commit as 0.2.3, so it should appear in melpa-stable soon.

**Comment by kasz** [comment]
Yes, I was referring to melpa-stable. Thank you for fast answer.

**Comment by MalloZup** [comment]
autogenerated with https://github.com/MalloZup/doghub:  issue inactive since 450 days. Please update the issue or close it
